pkgs/development/python-modules/python-jwt/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, cryptography
+, freezegun
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "jwt";
+  version = "1.3.1";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    inherit version;
+    owner = "GehirnInc";
+    repo = "python-jwt";
+    rev = "v${version}";
+    sha256 = "sha256-N1J8yBVX/O+92cRp+q2gA2cFsd+C7JjUR9jo0VGoINg=";
+  };
+
+  propagatedBuildInputs = [
+    cryptography
+    freezegun
+    pytest
```

Comment:
`cryptography` should be the only propagatedBuildInputs: https://github.com/GehirnInc/python-jwt/blob/068db420c9ae957925daf0f5a2baa9319ac20c82/setup.py#L27

The rest might find their place in nativeCheckInputs (also there is pytestCheckHook)

Reply:
Good call. I tried using pytestCheckHook but it threw usage errors. Instead of diving into it I just added them to nativeCheckInputs